### PR TITLE
New Minionese Cheat Sheet

### PR DESCRIPTION
**Pull request by swapagarwal**
IA: https://duck.co/ia/view/minionese_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/minionese.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "minionese_cheat_sheet",
+    "name": "Minionese Cheat Sheet",
+    "description": "A List Of Minion Words And Their English Translations",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/minionese.json]: @swapagarwal Lets avoid capitalisation of first lettrs of each word.

File: share/goodie/cheat_sheets/json/language/minionese.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                "key": "Chair",
+                "val": "Chasy"
+            },
+            {
+                "key": "Cheers",
+                "val": "Kampai"
+            },
+            {
+                "key": "Fire",
+                "val": "BEE DO BEE DO BEE DO"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/minionese.json]: @swapagarwal Why is it all capal letters?

**Comment by GuiltyDolphin** [comment]
@swapagarwal Don't worry the failing tests have nothing to do with this cheat sheet.

@moollaza SunInfo is now failing, you recently tweaked it, could you have a quick check over your changes?

File: share/goodie/cheat_sheets/json/language/minionese.json (hunk 1 above)
Comment (swapagarwal) [inline share/goodie/cheat_sheets/json/language/minionese.json]: Sure.

File: share/goodie/cheat_sheets/json/language/minionese.json (hunk 2 above)
Comment (swapagarwal) [inline share/goodie/cheat_sheets/json/language/minionese.json]: It was for a siren effect, but now I think "Bee do" will do just fine.

Comment (jophab) [inline share/goodie/cheat_sheets/json/language/minionese.json]: @swapagarwal  :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=minion language cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/minionese_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=minion language cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/minionese_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=minion language cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/minionese_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=minion language cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/minionese_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@swapagarwal apologies for the delay! This LGTM :+1:

**Comment by swapagarwal** [comment]
No problem. :smile:

**Pull request merged**